Modules/Sources/NetworkClient/Client.swift

```diff
@@ -26,53 +25,60 @@ extension NetworkClient.NetworkPath {
     }
 }
 
+#if DEBUG
 extension NetworkClient {
-    public static let failing = Self.init(pathUpdatePublisher: .failing())
+    public static var failing: Self {
+        return .init(pathUpdateStream: {
+            XCTFail("\(Self.self).pathUpdateStream method not implemented.")
+            fatalError("\(Self.self).pathUpdateStream method not implemented.")
+        })
```

Comment:
```suggestion
        return .init(pathUpdateStream: {
            XCTUnimplemented("\(Self.self).pathUpdateStream method not implemented.")
        })
```

Reply:
I've just refactored our implementation to use async/await too and the above was suggested to me by Jakob so now I'm passing it on 😄